RUNNING_INFERSHARP_ON_WINDOWS.md

```diff
@@ -0,0 +1,40 @@
+# How to Run Infer# Locally on Windows via WSL2 (Windows Subsystem for Linux)
+
+## Prerequisite
+- [Enable WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+*We recommend using Ubuntu or Debian WSL distribution.*
```

Comment:
We recommend using the WSL distribution for either Ubuntu or Debian.

Reply:
WSL has several different Linux distros, Ubuntu and Debian are the two we recommend.

Reply:
I understand that -- I'm just commenting that the wording you have is a bit weird.